src/main/java/meteordevelopment/meteorclient/mixin/DisconnectedScreenMixin.java

```diff
@@ -44,11 +45,15 @@ private void onRenderBackground(CallbackInfo info) {
                 button -> ConnectScreen.connect(new MultiplayerScreen(new TitleScreen()), client,
                 ServerAddress.parse(Modules.get().get(AutoReconnect.class).lastServerInfo.address),
                 Modules.get().get(AutoReconnect.class).lastServerInfo)));
-            toggleAutoReconnectBtn = addDrawableChild(new ButtonWidget(x, y + 22, 200, 20, Text.literal("Toggle AutoReconnect"),
+            addDrawableChild(new ButtonWidget(x, y + 22, 200, 20, Text.literal("Toggle AutoReconnect"),
                 button -> {
                     Modules.get().get(AutoReconnect.class).toggle();
                     time = Modules.get().get(AutoReconnect.class).time.get() * 20;
                     }));
+            if (Modules.get().get(ServerSpoof.class).noSignatures()) addDrawableChild(new ButtonWidget(x, y + 44, 200, 20, Text.literal("Deactivate ServerSpoof & Reconnect"), button -> {
+                Modules.get().get(ServerSpoof.class).toggle();
```

Comment:
maybe just toggle no signatures to off

Reply:
No signatures has been removed so the button can also be removed.